pkg/protobuf/app/app.proto

```diff
@@ -17,6 +17,7 @@ message CreateRequest {
     string name = 1;
     string team = 2;
     string process_type = 3;
+    string virtual_host = 6;
```

Comment:
maybe? :
```diff
- 6
+ 4
```
🤔

Reply:
change numeric tags for any existing fields may break backward compatibility

Reply:
but `virtual_host` is a new field, right?

Reply:
yes but fields 4 and 5 already exists

Reply:
you're right :/ sorry